setup.sh

```diff
@@ -39,7 +40,7 @@ crontab -l > temp_crontab
 #echo new cron into cron file
 if [ "$(grep "${monitor_program_file}" temp_crontab)" == "" ]; then
     echo "*/30 * * * * python3 ${monitor_program_file}" >> temp_crontab
-    
+    echo "@reboot python3 ${check_restart_file}" >> temp_crontab
```

Comment:
The operation of `$monitor_program_file` and `$check_restart_file` is very similar, but it can not only detect one keyword to determine that the `temp_crontab` needs to be updated, but it will also make an upgrade error.
Use another if-else to detect whether `$check_restart_file` is in the `temp_crontab`.